tests/tests_fabric/conftest.py

```diff
@@ -72,9 +72,18 @@ def teardown_process_group():
 def reset_deterministic_algorithm():
     """Ensures that torch determinism settings are reset before the next test runs."""
     yield
+    os.environ.pop("CUBLAS_WORKSPACE_CONFIG", None)
     torch.use_deterministic_algorithms(False)
 
 
+@pytest.fixture
+def reset_cudnn_benchmark():
+    """Ensures that the `torch.backends.cudnn.benchmark` setting gets reset before the next test runs."""
+    benchmark = torch.backends.cudnn.benchmark
+    yield
+    torch.backends.cudnn.benchmark = benchmark
```

Comment:
Shouldn't this be

```suggestion
    yield
    torch.backends.cudnn.benchmark = False
```


We don't want to leave benchmark set.

Also in `reset_deterministic_algorithms` we force False too

Reply:
reset_deterministic_algorithms should reset the old value too :(